spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -384,6 +384,11 @@ public Template getTemplate() {
 		 */
 		private String keyStorePassword;
 
+		/**
+		 * Key stroe algorithm.
+		 */
+		private String keyStoreAlgorithm;
```

Comment:
This field's initial value should be `"SunX509"`, aligning it with Spring AMQP's default.